src/cli/factories/commandFactory.ts

```
@@ -3019,23 +3029,43 @@ export class CLICommandFactory {
+        }
+        abortHandler.signal.addEventListener(
+          "abort",
+          () => {
+            reject(new DOMException("Stream aborted", "AbortError"));
+          },
+          { once: true },
+        );
+      });
+
       while (true) {
         let nextResult;
```

Biome (2.4.10)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)